README.md

```diff
@@ -250,6 +250,16 @@ To build LPVS from source code and run it, follow these steps:
    ```
    > Note: Use `LPVS` as the value for the `-Dgithub.secret=` parameter.
 
+   Also, available configuration to run single scan on pull request, using following command: 
```

Comment:
This is a new scenario. Please create a separate section with proper description.